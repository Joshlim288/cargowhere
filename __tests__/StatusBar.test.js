/**
 * @format
 */
import React from 'react';
import StatusBar from '../StatusBar';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('StatusBar renders correctly', () => {
  const tree = renderer.create(<StatusBar />).toJSON();
  expect(tree).toMatchSnapshot();
});
