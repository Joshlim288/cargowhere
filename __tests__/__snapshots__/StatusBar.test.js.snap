// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": undefined,
      "height": 20,
    }
  }
/>
`;
